samtranslator/model/sam_resources.py

```diff
@@ -2111,23 +2116,36 @@ def _add_connector_metadata(
         original_resources = original_template.get("Resources", {})
         original_source_type = original_resources.get(source.logical_id, {}).get("Type")
         original_dest_type = original_resources.get(destination.logical_id, {}).get("Type")
-        metadata = {
-            "aws:sam:connectors": {
-                self.logical_id: {
-                    # If the source/destination is a serverless resource,
-                    # we prefer to include the original serverless resource type
-                    # over the transformed CFN resource type so it can distinguish
-                    # connector usage between serverless resources and CFN resources.
-                    "Source": {"Type": original_source_type or source.resource_type},
-                    "Destination": {"Type": original_dest_type or destination.resource_type},
-                }
-            }
-        }
+
         for resource in generated_resources:
             # Although as today the generated resources do not have any existing metadata,
             # To make it future proof, we still does a merge to avoid overwriting.
             try:
                 original_metadata = resource.get_resource_attribute("Metadata")
             except KeyError:
                 original_metadata = {}
+
+            connector_type = STANDARD_CONNECTOR
+            if "aws:sam:embedded:connector" in original_metadata and is_multi_dest:
```

Comment:
Would it be worth making `connector_type` a list and adding `EMBEDDED_CONNECTOR` and `MULTIDEST_CONNECTOR` rather than making a separate `EMBEDDED_MULTIDEST_CONNECTOR` type? It'd be one less type

Reply:
I'll actually make them a boolean value, like Chris suggested.